Ansible doesn't provision properly at the app_server : npm step

Seems like it didn't become sudo... on Mac OS X 10.11.5, ansible 2.1.0.0, vagrant 1.8.4
```
TASK [app_server : npm] ********************************************************
fatal: [default]: FAILED! => {"changed": false, "cmd": "/usr/bin/npm install --global ''", "failed": true, "msg": "npm ERR! Error:********@3.6.0\"\nnpm ERR! cwd /home/vagrant\nnpm ERR! node -v v0.10.37\nnpm ERR! npm -v 1.4.28\nnpm ERR! path /usr/bin/npm\nnpm ERR! code EACCES\nnpm ERR! errno 3\nnpm ERR! stack Error: EACCES, unlink '/usr/bin/npm'\nnpm ERR! error rolling back Error: EACCES, unlink '/usr/bin/npm'\nnpm ERR! error rolling back  { [Error: EACCES, unlink '/usr/bin/npm'] errno: 3, code: 'EACCES', path: '/usr/bin/npm' }\nnpm ERR! not ok code 0", "rc": 3, "stderr": "npm ERR! Error: EACCES, unlink '/usr/bin/npm'\nnpm ERR!  { [Error: EACCES, unlink '/usr/bin/npm'] errno: 3, code: 'EACCES', path: '/usr/bin/npm' }\nnpm ERR! \nnpm ERR! Please try running this command again as root/Administrator.\n\nnpm ERR! System Linux 3.13.0-91-generic\nnpm ERR! command \"/usr/bin/node\" \"/usr/bin/npm\" \"install\" \"--global\" \"npm@3.6.0\"\nnpm ERR! cwd /home/vagrant\nnpm ERR! node -v v0.10.37\nnpm ERR! npm -v 1.4.28\nnpm ERR! path /usr/bin/npm\nnpm ERR! code EACCES\nnpm ERR! errno 3\nnpm ERR! stack Error: EACCES, unlink '/usr/bin/npm'\nnpm ERR! error rolling back Error: EACCES, unlink '/usr/bin/npm'\nnpm ERR! error rolling back  { [Error: EACCES, unlink '/usr/bin/npm'] errno: 3, code: 'EACCES', path: '/usr/bin/npm' }\nnpm ERR! not ok code 0\n", "stdout": "", "stdout_lines": []}
	to retry, use: --limit @provisioning/vagrant.retry
```

--- a/provisioning/roles/app_server/tasks/install_dependencies.yml
+++ b/provisioning/roles/app_server/tasks/install_dependencies.yml
@@ -1,7 +1,6 @@
 ---
 - command: bower install chdir={{ project_dir }} --allow-root
 - composer: working_dir={{ project_dir }}
-- npm: name=npm version=3.6.0 global=yes
 - npm: path={{ project_dir }}
 - npm: path={{ project_dir }}/api
 - git:

--- a/provisioning/roles/app_server/tasks/main.yml
+++ b/provisioning/roles/app_server/tasks/main.yml
@@ -5,6 +5,7 @@
     - php5-curl
     - php5-mysql
     - python-mysqldb  # Required for mysql ansible modules
+- npm: name=npm version=3.6.0 global=yes
 - { include: install_dependencies.yml, become: yes, become_user: "{{ user }}" }
 - cron: name='Update API data' special_time=hourly job='cd {{ project_dir }}/api && grunt default rsync --target=dist' user={{ user }}
 - mysql_db: name={{ yourls_db_name }}

--- a/provisioning/roles/nodejs/tasks/main.yml
+++ b/provisioning/roles/nodejs/tasks/main.yml
@@ -1,3 +1,5 @@
 ---
-- apt_repository: repo=ppa:chris-lea/node.js
+- get_url: url=https://deb.nodesource.com/setup_5.x dest={{ project_dir }}/setup_5.x mode=0744
+- shell: "{{ project_dir }}/setup_5.x"
+  become: yes
 - apt: name=nodejs

